iOS/DuckDuckGo/AIChat/InputBox/SwitchBar/NavigationActionBarView.swift

```diff
@@ -110,12 +110,14 @@ final class NavigationActionBarView: UIView {
         mainStackView.translatesAutoresizingMaskIntoConstraints = false
         backgroundGradientView.translatesAutoresizingMaskIntoConstraints = false
         
+        let mainStackMinHeightConstraint = mainStackView.heightAnchor.constraint(greaterThanOrEqualToConstant: Constants.buttonSize)
```

Comment:
Required to prevent the shadow from animating when transitioning from a state without any buttons to a state with buttons